app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -299,6 +299,27 @@ def update_recordings
     render(:update_recordings)
   end
 
+  def delete_recordings
+    raise BBBError.new('missingParamRecordID', 'You must specify a recordID.') if params[:recordID].blank?
+
+    Recording.transaction do
+      query = Recording.where(record_id: params[:recordID].split(','))
+      raise BBBError.new('notFound', 'We could not find recordings') if query.none?
+
+      query.each do |rec|
```

Comment:
Right now the `query.none?` and `query.each` will do two separate database calls. You can fix this by manually indicating when to load the data: Add `.load` to the end of the `query = ` line.